filecoin-proofs/tests/api.rs

```diff
@@ -1311,6 +1323,11 @@ fn window_post<Tree: 'static + MerkleTreeTrait>(
 
     let mut vanilla_proofs = Vec::with_capacity(replica_sectors.len());
 
+    // Make sure that files can be read-only for a window post.
+    for (_, _, _, cache_dir, _) in &sectors {
+        set_readonly_flag(cache_dir.path(), true);
```

Comment:
I would also apply read-only setting to the replica file besides cache directory